2020-03-07 13:35:27 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2020-03-07 13:35:35 jdbc[5]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2020-03-07 13:49:52 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2020-03-07 13:51:29 jdbc[7]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.getAllCids(JkuserDaoImpl.java:810)
	at com.qq.server.ChatTools.sendOnOffLineMsg(ChatTools.java:130)
	at com.qq.server.ChatTools.addClient(ChatTools.java:81)
	at com.qq.server.ServerThread.run(ServerThread.java:188)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.regUser(JkuserDaoImpl.java:30)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:103)
	at com.qq.server.ServerThread.run(ServerThread.java:183)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2020-03-07 13:51:29 jdbc[7]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.regUser(JkuserDaoImpl.java:30)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:103)
	at com.qq.server.ServerThread.run(ServerThread.java:183)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2020-03-07 13:51:40 jdbc[9]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.regUser(JkuserDaoImpl.java:30)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:103)
	at com.qq.server.ServerThread.run(ServerThread.java:183)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2020-03-07 13:52:23 jdbc[11]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.getAllCids(JkuserDaoImpl.java:810)
	at com.qq.server.ChatTools.sendOnOffLineMsg(ChatTools.java:130)
	at com.qq.server.ChatTools.addClient(ChatTools.java:81)
	at com.qq.server.ServerThread.run(ServerThread.java:188)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.BaseJdbcDao.findById(BaseJdbcDao.java:155)
	at com.qq.server.ChatTools.sendMsg2One(ChatTools.java:228)
	at com.qq.server.ServerThread.run(ServerThread.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2020-03-07 13:52:23 jdbc[11]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.checkLogin(JkuserDaoImpl.java:99)
	at com.qq.server.ServerThread.checkLogin(ServerThread.java:171)
	at com.qq.server.ServerThread.readFirstMsg(ServerThread.java:78)
	at com.qq.server.ServerThread.run(ServerThread.java:183)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.BaseJdbcDao.findById(BaseJdbcDao.java:155)
	at com.qq.server.ChatTools.sendMsg2One(ChatTools.java:228)
	at com.qq.server.ServerThread.run(ServerThread.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2020-03-07 13:53:32 jdbc[16]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.BaseJdbcDao.findById(BaseJdbcDao.java:155)
	at com.qq.server.ChatTools.sendMsg2One(ChatTools.java:228)
	at com.qq.server.ServerThread.run(ServerThread.java:192)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.BaseJdbcDao.save(BaseJdbcDao.java:43)
	at com.qq.server.ChatTools.sendMsg2One(ChatTools.java:278)
	at com.qq.server.ServerThread.run(ServerThread.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 13 more
2020-03-07 13:57:09 jdbc[21]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.updateIcon(JkuserDaoImpl.java:725)
	at com.qq.server.ChatTools.sendMsg2One(ChatTools.java:202)
	at com.qq.server.ServerThread.run(ServerThread.java:192)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.getAllCids(JkuserDaoImpl.java:810)
	at com.qq.server.ChatTools.sendOnOffLineMsg(ChatTools.java:130)
	at com.qq.server.ChatTools.removeClient(ChatTools.java:95)
	at com.qq.server.ServerThread.run(ServerThread.java:199)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2020-03-07 13:57:09 jdbc[21]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.updateIcon(JkuserDaoImpl.java:725)
	at com.qq.server.ChatTools.sendMsg2One(ChatTools.java:202)
	at com.qq.server.ServerThread.run(ServerThread.java:192)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.qq.util.JdbcUtil.getConnection(JdbcUtil.java:31)
	at com.qq.dao.JkuserDaoImpl.getAllCids(JkuserDaoImpl.java:810)
	at com.qq.server.ChatTools.sendOnOffLineMsg(ChatTools.java:130)
	at com.qq.server.ChatTools.removeClient(ChatTools.java:95)
	at com.qq.server.ServerThread.run(ServerThread.java:199)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
